tablecodec/tablecodec.go

```diff
@@ -123,7 +123,7 @@ func hasRecordPrefixSep(key kv.Key) bool {
 
 // DecodeRecordKey decodes the key and gets the tableID, handle.
 func DecodeRecordKey(key kv.Key) (tableID int64, handle kv.Handle, err error) {
-	if len(key) <= prefixLen {
+	if len(key) < prefixLen {
```

Comment:
If this is changed, we may get an empty byte slice common handle for the key like `t{table_id}_r`. I think it is better to return a `nil` value?